github/pulls_test.go

```diff
@@ -97,23 +97,40 @@ func TestPullRequestsService_Get_headAndBase(t *testing.T) {
 	}
 }
 
-func TestPullRequestService_Get_DiffURLAndPatchURL(t *testing.T) {
+func TestPullRequestsService_Get_URLFields(t *testing.T) {
```

Comment:
Minor, but it looks like the pattern for test name we're using is as described for examples [here](https://godoc.org/testing#hdr-Examples).

> Multiple example functions for a package/type/function/method may be provided by appending a distinct suffix to the name. **The suffix must start with a lower-case letter.**
>
> ```Go
> func Example_suffix() { ... }
> func ExampleF_suffix() { ... }
> func ExampleT_suffix() { ... }
> func ExampleT_M_suffix() { ... }
> ```

So according to that, the suffix `_URLFields` should be `_urlFields`.

It would be more consistent with the rest of the tests here, such as `TestPullRequestsService_Get_headAndBase` and `TestPullRequestsService_Get_invalidOwner`.